tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -56,44 +58,75 @@ def lambda_handler(event, context):
         AND
             status_id = %d
     """
-    #Gets the dbconnection info.
+    # Gets the dbconnection info.
 
-    db_connect_info = requests_db.get_dbconnect_info()
+    db_connect_info = get_dbconnect_info()
 
     try:
-        rows = database.single_query(sql, database.db_connect_info, (filename, shared_recovery.OrcaStatus.PENDING))
-    except database.DbError as err:
-        LOGGER.error(f"DbError: {str(err)}")
-        raise DatabaseError(str(err))
+        rows = single_query(sql, db_connect_info, (filename, OrcaStatus.PENDING.value))
+    except Exception as ex:
+        LOGGER.error(ex)
 
     if len(rows) == 0:
-        return None
-    orca_recoveryfile = rows[0]
-    # grab the parameters from the db in json format
-    job_id = database.result_to_json(orca_recoveryjob)['job_id']
-    granule_id = database.result_to_json(orca_recoveryjob)['granule_id']
-    restore_destination = database.result_to_json(orca_recoveryjob)['restore_destination']
-
-    #post to staged-recovery-queue
-    shared_recovery.post_status_for_file_to_queue(
-        job_id = job_id,
-        granule_id = granule_id,
-        filename = filename,
-        restore_destination = restore_destination,
-        status_id = shared_recovery.OrcaStatus.STAGED,
-        error_message = '',
-        request_method = shared_recovery.RequestMethod.NEW,
-        db_queue_url= OS_ENVIRON_RECOVERY_QUEUE_URL_KEY
+        raise Exception(
+            "length of rows cannot be empty since there will always be a record in the database"
         )
+    # convert db result to json
+    db_result_json = result_to_json(rows)
+
+    # grab the parameters from the db in json format
+    job_id = db_result_json["job_id"]
+    granule_id = db_result_json["granule_id"]
+    restore_destination = db_result_json["restore_destination"]
+
+    # post to staged-recovery-queue
+    post_status_for_file_to_queue(
```

Comment:
I don't think this is the correct call. This is for updating status in the database QUEUE. I believe you would want to use the base `post_entry_to_queue` call so that you could send the proper input needed for the copy to archive lambda. See:
- https://github.com/nasa/cumulus-orca/blob/develop/tasks/copy_files_to_archive/schemas/input.json
- https://github.com/nasa/cumulus-orca/blob/develop/tasks/copy_files_to_archive/schemas/sub_schemas/body.json

Reply:
According to the above schemas, these are the required fields needed:

`{
  "job_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
  "granule_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
  "filename": "ABCDEFGHIJ",
  "source_key": "ABCDEFGHIJK",
  "target_key": "ABCDEFGHIJKLMNOPQRSTUV",
  "restore_destination": "ABCDEFGHIJKLMNOPQRSTUVWXYZABC",
  "source_bucket": "ABCDEFGHIJKLMNOPQ"
}`

How to get the source_key, target_key? I have not seen them anywhere before.